spec/helpers/alchemy/elements_block_helper_spec.rb

```diff
@@ -162,7 +166,18 @@ module Alchemy
             mock_model("Content", essence: mock_model("EssenceText"))
           end
 
-          subject.essence :title
+          Alchemy::Deprecation.silence do
```

Comment:
Wondering if there is a way how to get reminded of those silencing blocks after the release? I just think otherwise it could be quite possible to forget about them and they'll stay forever (or until someone works on this file / line of code and notices its obsolete).

Reply:
Very fair point. Usually I go over all deprecations before a new release, so it's a manual process anyway.